File: app_go/internals/handlers/time.go
package handlers

import (
	"fmt"
	"log"
	"net/http"
	"time"
)

type TimeHandler struct{}

func NewTimeHandler() *TimeHandler {
	return &TimeHandler{}
}

func (t *TimeHandler) CurrentTime(w http.ResponseWriter, r *http.Request) {
	timeZone := "Europe/Moscow"
	location, err := time.LoadLocation(timeZone)

	if err != nil {
		err_text := fmt.Sprintf("load location: %v", err)
		log.Print(err_text)
		fmt.Fprintf(w, err_text)
	} else {
		currentTime := time.Now().In(location)
		fmt.Fprintf(w, "Current Time: %s", currentTime.Format(time.DateTime))
	}
}

File: app_python/app/utils/time.py
"""This module contains helpful functions to time"""

from datetime import datetime
import pytz


def moscow_timezone():
    """returns moscow timezone"""
    return pytz.timezone("Europe/Moscow")

def moscow_time():
    """returns moscow time"""

    current_moscow_time = datetime.now(moscow_timezone())

    return current_moscow_time
